mysql-test/suite/galera/t/galera_binlog_stmt_autoinc.test

```diff
@@ -0,0 +1,84 @@
+##
+## Tests the auto-increment with binlog in STATEMENT mode.
+##
+
+--source include/galera_cluster.inc
```

Comment:
Use-case involves async master with one of galera node from galera cluster acting as slave.

I see you have only galera cluster involved. How is this testing the said use-case.